pkg/agent/apiserver/apiserver.go

```diff
@@ -131,6 +132,10 @@ func newConfig(npq querier.AgentNetworkPolicyInfoQuerier, bindPort int, enableMe
 	secureServing.BindAddress = net.IPv4zero
 	secureServing.BindPort = bindPort
 
+	// With AntreaProxy enabled, it is possible that K8s Service flows and routes are not ready when apiserver tries to
+	// load configmap based request-header-client-ca-file.
+	authentication.ClientTimeout = 60 * time.Second
```

Comment:
what's the default value, and why there isn't issue on Linux?

Reply:
The default value is 10 seconds. I not sure why nothing happens on Linux but I will retry after the rebase.

Reply:
(Please don't resolve the comment before we get a conclusion) I'm concerned the 6x timeout, which might cause more requests and goroutines stuck when network goes wrong. 
If you don't change it, what would be the worst case? Not being able to process incoming requests before AntreaProxy installs flows?

Reply:
It crashes.
```
F0928 02:23:25.524737   18076 main.go:58] Error running agent: error when creating agent API server: unable to load configmap based request-header-client-ca-file: Get "https://10.96.0.1:443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication?timeout=10s": context deadline exceeded
goroutine 1 [running]:
k8s.io/klog/v2.stacks(0x1)
        C:/go/pkg/mod/k8s.io/klog/v2@v2.8.0/klog.go:1021 +0x8a
k8s.io/klog/v2.(*loggingT).output(0x3384ea0, 0x3, {0x0, 0x0}, 0xc00090c310, {0x274ce54, 0x1}, 0xc000adcaf0, 0x0)
        C:/go/pkg/mod/k8s.io/klog/v2@v2.8.0/klog.go:970 +0x569
k8s.io/klog/v2.(*loggingT).printf(0xc00028c100, 0x887da8, {0x0, 0x0}, {0x0, 0x0}, {0x1ea00f0, 0x17}, {0xc000adcaf0, 0x1, ...})
        C:/go/pkg/mod/k8s.io/klog/v2@v2.8.0/klog.go:751 +0x1d1
k8s.io/klog/v2.Fatalf(...)
        C:/go/pkg/mod/k8s.io/klog/v2@v2.8.0/klog.go:1509
main.newAgentCommand.func1(0xc000202840, {0xc0002226c0, 0x0, 0x6})
        C:/antrea/cmd/antrea-agent/main.go:58 +0x28e
github.com/spf13/cobra.(*Command).execute(0xc000202840, {0xc0000bc010, 0x6, 0x7})
        C:/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0xc000202840)
        C:/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        C:/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
        C:/antrea/cmd/antrea-agent/main.go:37 +0x4a
```